x-pack/plugins/ml/public/maps/layer_selector.tsx

```diff
@@ -36,7 +36,7 @@ export class LayerSelector extends Component<Props, State> {
     const typicalActual: MlAnomalyLayers = selectedOptions[0].value! as
       | 'typical'
```

Comment:
Would be good to have these as a constant so they can be used in `getSupportedShapeTypes` and other places as well

Reply:
Added in 2321384960d2018fc3aa3e54ddf0d4e5c95ae80a